pkg/kubefedctl/enable/enable.go

```diff
@@ -282,6 +282,36 @@ func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResou
 	} else if err != nil {
 		return errors.Wrapf(err, "Error getting CRD %q", resources.CRD.Name)
 	} else {
+		ftcs := &fedv1b1.FederatedTypeConfigList{}
+		err := client.List(context.TODO(), ftcs, namespace)
+		if err != nil {
+			return errors.Wrap(err, "Error getting FederatedTypeConfig list")
+		}
+
+		for _, obj := range ftcs.Items {
+			ftc := &fedv1b1.FederatedTypeConfig{}
+			err = client.Get(context.TODO(), ftc, namespace, obj.Name)
```

Comment:
This client call does not appear to be necessary since `obj` is already the desired ftc.